app/views/layouts/application.html.erb

```diff
@@ -90,24 +90,24 @@
   <div class="container">
     <hr>
 
-    <div class="row">
+    <div id="footer-authoring" class="row">
       <div class="col-md-12">
         <%= yield :authoring %>
       </div>
     </div>
 
-    <div class="row">
+    <div id="footer-copyright" class="row">
       <div class="col-md-4 text-left">
         <p>&copy; Copyright 2015-<%= DateTime.now.year %>
           <a href="http://mumuki.org/" class="mu-org-link"><span class="da da-mumuki-circle"></span> Mumuki Project</a>
         </p>
       </div>
 
-      <div class="col-md-4 text-center">
+      <div id="footer-auth0" class="col-md-4 text-center">
```

Comment:
It is not necessary auth0, it is provider-dependent. In fact, when using SAML, no content is displayed within this div.